src/main/java/org/spdx/library/model/ModelObject.java

```diff
@@ -40,13 +40,8 @@
 import org.spdx.library.model.enumerations.ReferenceCategory;
 import org.spdx.library.model.enumerations.RelationshipType;
 import org.spdx.library.model.enumerations.SpdxEnumFactory;
-import org.spdx.library.model.license.AnyLicenseInfo;
-import org.spdx.library.model.license.ConjunctiveLicenseSet;
+import org.spdx.library.model.license.*;
```

Comment:
Based on the existing codebase, I _think_ @goneall prefers explicit imports, rather than wildcard imports.  FWIW I'd be in favour of using wildcard imports though, if he's open to that.

Reply:
Fair point about keeping style with the imports. Do you reckon I should just change history and force-push this fix?

Reply:
I'm OK with the wildcards and the minor style inconsistency in using a mix of specific imports and wildcard.  I have my IDE set to generate the specific imports - mainly to avoid possible conflicts but in this case the public classes shouldn't create any conflicts.